content/22.vaccines.md

```diff
@@ -669,9 +678,14 @@ Additionally, Germany has stated that if Sputnik V were approved by EMA, it woul
 Russia is developing other vaccine candidates and has approved a third vaccine, CoviVac, which is an inactivated vaccine produced by the Chumakov Centre in Moscow, despite the fact the clinical trials have yet to begin [@url:https://www.reuters.com/article/us-health-coronavirus-russia-vaccine-idUSKBN2AK07H].
 
 In Asia, China and India are the main COVID-19 vaccination developers and providers.
+Thus far, over 63 million people have been vaccinated against COVID-19 vaccine in India [@url:https://www.mygov.in/covid-19/].
+A broad range of COVID-19 vaccine candidates are under investigation in order to respond to the COVID-19 pandemic.
 In India, the Covaxin vaccine produced by Bharat Biotech received emergency authorization on January 3rd, 2021, despite the lack of phase III data until March 3rd [@url:india-covid-19-vaccine.html; @url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf].
```

Comment:
Is it usually written as just Covaxin, or in all caps? Just wondering for consistency :)

Reply:
Wherever I have seen it, it is in all CAPS